cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -24,10 +24,13 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 // CNV-4134 virtctl upload
 
 // CNV-4080 VMI exposed in UI
+* You can view and manage virtual machine instances that are independent of virtual machines within the web console.
 
 // CNV-4079 CD-ROM added to VM wizard
+* {CNVProductNameStart} enables you to configure a CD-ROM in the virtual machine wizard. You can select the type of CD-ROM configuration (Container, URL, Persistent Volume Claim) from a drop-down list in the *Edit CD-ROM* window.
 
 // CNV-4075 VM boot order
+* The boot order list is now available in the virtual machine wizard. You can add, edit, and remove disks or Network Interface Cards in the virtual machine wizard.
```

Comment:
It is available in the VM details actually. It is not available in the wizard.

Reply:
Updated the content.